File: app/src/main/java/com/student/student_healthy/Data/MyApplication.java
package com.student.student_healthy.Data;

import android.app.Application;

import com.firebase.client.Firebase;
import com.google.firebase.FirebaseApp;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        FirebaseApp.initializeApp(this);
        Firebase.setAndroidContext(this);

    }
}

File: app/src/main/java/com/student/student_healthy/MyApplication.kt
package com.student.student_healthy

import android.app.Application;

import com.firebase.client.Firebase;
import com.google.firebase.FirebaseApp;

class  MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        FirebaseApp.initializeApp(this);
        Firebase.setAndroidContext(this);


    }


}